src/ParserConfig.cc

```diff
@@ -40,7 +40,7 @@ class sdf::ParserConfig::Implementation
     EnforcementPolicy::WARN;
 
   /// \brief Policy indicating how deprecated elements are treated.
-  /// Defaults to the value of `warningsPolicy`. It can be overriden by the user
+  /// Defaults to the value of `warningsPolicy`. It can be overridden by the user
```

Comment:
```suggestion
  /// Defaults to the value of `warningsPolicy`, can be overridden by the user
```
one way to reduce line length to fix the linter complaining in github workflow

Reply:
your fix in https://github.com/gazebosim/sdformat/pull/1552/commits/e9b9440d803e560918a2e5cf682c4523a84d8257 is a fine alternative